cypress/e2e/WebInterface/Measure/EditMeasure/EditMeasureOwnershipValidations.cy.ts

```diff
@@ -36,9 +36,13 @@ describe('Read only for measure, measure group, and test cases that user does no
         OktaLogin.Login()
     })
 
-    afterEach('Logout', () => {
+    afterEach('Logout and clean up', () => {
+
+        //Log out
```

Comment:
should we enable logout in this case? is this related to the cookie issue?

Reply:
yes this is related to cookie issue. Added new function for logout.